### PR TITLE
Update Sonarr-Menu to v3.1 and fix file name

### PR DESCRIPTION
**Pull request by jefbarn**

### CODE REVIEW

File: Casks/sonarr-menu.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    file = File.read(sonarr_path)
+    file = file.gsub(/^.*Sonarr-Menu.*$\r?\n/, '')
+
+    # Add new hook
+    file = file.sub(/MONO_EXEC=/, "#{hook}\n\\0")
+    File.write(sonarr_path, file)
+  end
+
+  zap :delete => '~/Library/LaunchAgents/SonarrAgent.plist'
+
```
Comment (vitorgalvao) [inline Casks/sonarr-menu.rb]: This empty line must go.

File: Casks/sonarr-menu.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  name 'Sonarr Menu'
+  homepage 'https://github.com/jefbarn/Sonarr-Menu'
+  license :gpl
+
+  app 'Sonarr-Menu.app'
+
+  depends_on :cask => 'sonarr'
+  depends_on :macos => '>= 10.9'
+
+  postflight do
```
Comment (vitorgalvao) [inline Casks/sonarr-menu.rb]: What exactly is this app, and why does it need this?

Comment (jefbarn) [inline Casks/sonarr-menu.rb]: This app provides an OSX user interface extension to the Sonarr app.  The postflight stanza adds a hook to the Sonarr app launch script to also launch the Menu at the same time.

File: Casks/sonarr-menu.rb (hunk 1 above)
Comment (jefbarn) [inline Casks/sonarr-menu.rb]: fixed

File: Casks/sonarr-menu.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/sonarr-menu.rb]: That part is **unacceptable**. We’ll not simply change another app on the behalf of yours. If you want to do that, make your app do it and warn your users, homebrew-cask won’t do such thing.

File: Casks/sonarr-menu.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    # Remove old hook
+    file = File.read(sonarr_path)
+    file = file.gsub(/^.*Sonarr-Menu.*$\r?\n/, '')
+
+    # Add new hook
+    file = file.sub(/MONO_EXEC=/, "#{hook}\n\\0")
+    File.write(sonarr_path, file)
+  end
+
+  zap :delete => '~/Library/LaunchAgents/SonarrAgent.plist'
```
Comment (vitorgalvao) [inline Casks/sonarr-menu.rb]: This isn’t a `zap`, it is very much an `uninstall`.

File: Casks/sonarr-menu.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+cask :v1 => 'sonarr-menu' do
+  version 'v3.1'
```
Comment (vitorgalvao) [inline Casks/sonarr-menu.rb]: `v` shouldn’t be here.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in f50143c370f74c7d5d6d94bc36663c70028c4343. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/sonarr-menu.rb (hunk 2 above)
Comment (jefbarn) [inline Casks/sonarr-menu.rb]: I'm good with this, it seemed a little hackey to me also. Tunnel vision I guess.

File: Casks/sonarr-menu.rb (hunk 3 above)
Comment (jefbarn) [inline Casks/sonarr-menu.rb]: Cool, I didn't know uninstall had all those options.  I like your change.

**Pull request closed without merging**